src/sentry/grouping/enhancer/__init__.py

```diff
@@ -321,7 +321,7 @@ def get_matching_frame_actions(self, frames, platform, exception_data=None, cach
 
         # 1 - Check if exception matchers match
         for m in self._exception_matchers:
-            if not m.matches_frame(frames, -1, platform, exception_data, cache):
+            if not m.matches_frame(frames, None, platform, exception_data, cache):
```

Comment:
The intend was always to pass an invalid frame index here, because exception matchers should not access the list of frames anyway.